profiles/chimay29hub.json

```diff
@@ -0,0 +1,6 @@
+{
+  "image": "https://user-images.githubusercontent.com/69494946/153624986-39c946b6-9bde-439f-bbb8-8c0ffc34286f.png",
```

Comment:
I recommend more of an image like this. What do you think?

![image](https://user-images.githubusercontent.com/51391473/153629567-d9f6f92c-e87d-4141-a9f8-04fab2707b2b.png)

```suggestion
  "image": "https://user-images.githubusercontent.com/51391473/153629567-d9f6f92c-e87d-4141-a9f8-04fab2707b2b.png",
```